commands/reminder/set-reminder.js

```diff
@@ -1,123 +1,103 @@
-const { Command } = require("discord.js-commando");
-const { DateTime } = require("luxon");
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { DateTime } = require('luxon');
 
-module.exports = class ReminderCommand extends Command {
-  constructor(client) {
-    super(client, {
-      name: "set-reminder",
-      aliases: ["reminder", "remind"],
-      group: "reminder",
-      memberName: "set-reminder",
-      description: "Sets a reminder",
-      userPermissions: ['MANAGE_CHANNELS'],
-      args: [
-        {
-          key: "deadline",
-          prompt: "When is the deadline in CST?",
-          type: "string",
-          validate: (deadline) => {
```

Comment:
i could not find a slash command equivalent so i moved the check inside the execute. let me know if you found a way. 😄